CopilotKit/packages/runtime/src/lib/runtime/copilot-runtime.ts

```diff
@@ -884,8 +884,8 @@ please use an LLM adapter instead.`,
   }
 
   discoverAgentsFromAgui(): Agent[] {
-    return Object.values(this.agents ?? []).map((agent: LangGraphAgent) => ({
-      name: agent.agentName,
+    return Object.entries(this.agents ?? []).map(([key, agent]: [string, AbstractAgent]) => ({
```

Comment:
Do we also need to patch the thread retrieval mechanism like in https://github.com/CopilotKit/CopilotKit/pull/2089? When I tested I noticed things would break on start-up and execution due to the client not being defined for non-LG systems.